update_remote_settings_records.mjs

```diff
@@ -28,45 +27,46 @@ const SUCCESS_RET_VALUE = 0;
 const FAILURE_RET_VALUE = 1;
 const VALID_ENVIRONMENTS = ["dev", "stage", "prod"];
 
-if (
-  !process.env.FX_REMOTE_SETTINGS_WRITER_USER ||
-  !process.env.FX_REMOTE_SETTINGS_WRITER_PASS
-) {
-  console.error(
-    `Both FX_REMOTE_SETTINGS_WRITER_USER and FX_REMOTE_SETTINGS_WRITER_PASS environment variables need to be set`
-  );
+if (!process.env.AUTHORIZATION) {
+  console.error(`AUTHORIZATION environment variable needs to be set`);
   process.exit(FAILURE_RET_VALUE);
 }
 
-if (!process.env.FX_REMOTE_SETTINGS_WRITER_SERVER) {
+if (!process.env.SERVER) {
   console.error(
-    `FX_REMOTE_SETTINGS_WRITER_SERVER environment variable needs to be set`
+    `SERVER environment variable needs to be set`
   );
   process.exit(FAILURE_RET_VALUE);
 }
 
 if (
-  process.env.FX_REMOTE_SETTINGS_ENVIRONMENT &&
+  process.env.ENVIRONMENT &&
   !VALID_ENVIRONMENTS.includes(process.env.ENVIRONMENT)
 ) {
   console.error(
-    `FX_REMOTE_SETTINGS_ENVIRONMENT environment variable needs to be set to one of the following values: ${VALID_ENVIRONMENTS.join(
+    `ENVIRONMENT environment variable needs to be set to one of the following values: ${VALID_ENVIRONMENTS.join(
       ", "
     )}`
   );
   process.exit(FAILURE_RET_VALUE);
 }
 
-const rsBrowsersCollectionEndpoint = `${process.env.FX_REMOTE_SETTINGS_WRITER_SERVER}/buckets/main-workspace/collections/devtools-compatibility-browsers`;
+const rsBrowsersCollectionEndpoint = `${process.env.SERVER}/buckets/main-workspace/collections/devtools-compatibility-browsers`;
 const rsBrowsersRecordsEndpoint = `${rsBrowsersCollectionEndpoint}/records`;
-const isDryRun = process.env.FX_REMOTE_SETTINGS_DRY_RUN == "1";
+const isDryRun = process.env.DRY_RUN == "1";
 
-const headers = {
-  "Content-Type": "application/json",
-  Authorization: `Basic ${btoa(
-    `${process.env.FX_REMOTE_SETTINGS_WRITER_USER}:${process.env.FX_REMOTE_SETTINGS_WRITER_PASS}`
-  )}`,
-};
+
+if (process.env.AUTHORIZATION.includes("Bearer")) {
+  const headers = {
+    "Content-Type": "application/json",
+    Authorization: process.env.AUTHORIZATION,
+  }
+} else {
+  const headers = {
+    "Content-Type": "application/json",
+    Authorization: `Basic ${btoa(${process.env.AUTHORIZATION})}`,
+  };
```

Comment:
In the docs, we wrote that it was supposed to receive the *raw* authorization value. But this seems more convenient, since you wouldn't have to do this `btoa()` step during the configuration of the job execution. 

We should update the docs then.